contracts/teller-network/Escrow.sol

```diff
@@ -146,13 +157,26 @@ contract Escrow is Pausable, MessageSigned {
      * @param _token Token address. Must be 0 for ETH
      * @param _tokenAmount How much ether/tokens will be put in escrow
      * @param _expirationTime Unix timestamp before the transaction is considered expired
+     * @param _statusContactCode The address of the status contact code
+     * @param _location The location on earth
+     * @param _username The username of the user
      * @dev Requires contract to be unpaused.
      *         The seller needs to be licensed.
      *         The expiration time must be at least 10min in the future
      *         For eth transfer, _amount must be equals to msg.value, for token transfer, requires an allowance and transfer valid for _amount
      */
-    function create_and_fund(address payable _buyer, address _token, uint _tokenAmount, uint _expirationTime, uint _tradeAmount, uint8 _tradeType) public payable whenNotPaused {
-        uint escrowId = create(_buyer, msg.sender, _token, _tradeAmount, _tradeType);
+    function create_and_fund(
+        address payable _buyer,
+        address _token,
+        uint _tokenAmount,
+        uint _expirationTime,
+        uint _tradeAmount,
+        uint8 _tradeType,
+        bytes memory _statusContactCode,
+        string memory _location,
+        string memory _username
```

Comment:
Buyer information won't be available here, no? Because this function is only called by the seller. Unless the seller gets the info for the buyer? Hmm, could make sense actually since they probably complete it together, but it's still a bit useless, since the username and contact code is only useful for distant comms.

What do you think?

Reply:
According to what we have now, this is the buyer creating the escrow and the seller funding it.

If a seller need to create an escrow then he will need information of the seller but it is not in our flow right now or I miss it.

I think we should do first minimum in order to get in work according to the flow we have and making sure we don't have problem with that already.
One problem I can see is that someone could change someone else username/location, which is a bit worrying me at the stage

Reply:
Well, my comment was specifically for `create_and_fund`, because that really is for the seller to call. 
And your point about changing another user really enforces my view that the seller shouldn't set the username and location of the buyer.

So, for `create`, the one that buyer calls, it's fine to set its infos, but `create_and_fund` should be only about the escrow

Reply:
I almost removed `create_and_fund` as it is not part of our use-cases
What do you think?

Reply:
Well, it's useful for when the buyer doesn't have ETH and can't create, so goes to a local seller and the seller creates the escrow for them.

I know we don't have a design for that yet, but I feel it can be really useful